src/views/partials/header.hbs

```diff
@@ -0,0 +1,71 @@
+<header>
+    <nav>
+        <div class="row-header">
+            <img src="/static/images/logo.png" alt="Shopping logo" class="logo_img">
+            <div class="main-nav">
+                <ul class="header-nav">
+                    <li>
+                        <a class="active" href="/">Home
+                        </a>
+                    </li>
+                    <li>
+                        <a href="product" class="hide">Product
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="super-header">
```

Comment:
You can follow Atomic design philosophy to organise your code more logically. It encourages re-usability and better code organisation